app/components/articleSearch/components/pagination/index.tsx

```diff
@@ -10,60 +10,56 @@ export interface IPaginationProps {
   searchQuery: string;
 }
 
-const Pagination = (props: IPaginationProps) => {
```

Comment:
If u want to refactor this code for clear meaning and organizing, you should divide whole rendering logic(especially if/else statements) to the separate functions. not just break props to variables.